modules/ROOT/pages/upgrade.adoc

```diff
@@ -32,6 +32,31 @@ Before migrating, ensure that you have performed and met the following prerequis
 
 * Ensure that your environment meets all of the system and network requirements described in xref:prereq-workflow.adoc[Workflow: Install and Verify Prerequisites for Physical Servers]
 
+* Ensure that the following kernel modules are enabled: https://docs.mulesoft.com/private-cloud/2.0/prereq-software#kernel-modules
```

Comment:
Is it necessary to repeat the content that's already in the other doc (following the link)?
I tend to keep content repetition to a minimum as it's hard to keep the content in sync.

Reply:
I wrestle with this - I agree with not duplicating info, but I also hate making users click a bunch of links. That said, I have removed the info and left the link.